docker-compose.yml

```diff
@@ -27,15 +27,12 @@ services:
       args:
         TZ: ${TZ}
     volumes:
-      - .:/code:delegated
-      # use the container's node_modules and bundles folder (don't override)
-      - /code/frontend/node_modules/
-      - /code/frontend/bundles/
-      - /code/static/
+      # Mount only the needed directories for local development
+      - ./frontend/components:/code/frontend/components
```

Comment:
Yeah, this makes sense to me to re-arrange this. It might be easier to do this on a separate PR after this one. It does reload components right now but I can see switching this over to having the reloaded code in it's own folder. 

It was either that or try to get node_modules somewhere else and that had it's own challenges.